static/css/misc.less

```diff
@@ -1030,24 +1030,23 @@
     - cfgov-misc
 */
 
+
+// Size of cicular background.
+@size: 1.5em;
```

Comment:
Or maybe just `@icon-size` ... I'm not sure whether and how units should appear in Less var names. Opinions?